### DISCUSSION TITLE
ScriptReference/AudioSettings-dspTime

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#395] [Notes]
# ScriptReference/AudioSettings-dspTime

https://docs.unity3d.com/ScriptReference/AudioSettings-dspTime.html

**Comment by Hunter-Bobeck** [comment]
It would seem pertinent to explain what "DSP" actually stands for. It stands for "[Digital Signal Processing](https://en.wikipedia.org/wiki/Digital_signal_processing)".
